azure/container-app/variables.tf

```diff
@@ -75,7 +75,7 @@ locals {
 variable "revision_mode" {
   type        = string
   description = "In Single mode, a single revision is in operation at any given time. In Multiple mode, more than one revision can be active at a time and can be configured with load distribution via the traffic_weight block in the ingress configuration."
-  default     = "Multiple"
+  default     = "Single"
```

Comment:
don't we need "Multiple" for zero-downtime deploys?

Reply:
```
In single revision mode, Container Apps ensures your app doesn't experience downtime when creating a new revision. The existing active revision isn't deactivated until the new revision is ready. 
```
https://learn.microsoft.com/en-us/azure/container-apps/revisions

Reply:
awesome! Can we add to the description, that single mode is enough for zero-downtime deployments?